compiler/circle-opselector/driver/Driver.cpp

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "SinglePass.h"
+#include "Function1.h"
+// TODO Add new pass headers
+
+#include <foder/FileLoader.h>
+
+#include <luci/Importer.h>
+#include <luci/CircleExporter.h>
+#include <luci/CircleFileExpContract.h>
+
+#include <arser/arser.h>
+#include <vconone/vconone.h>
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+void print_version(void)
+{
+  std::cout << "circle-opselector version " << vconone::get_string() << std::endl;
+  std::cout << vconone::get_copyright() << std::endl;
+}
+
+void split_id_input(const std::string &str, std::vector<int> &by_id)
+{
+  int input_length = str.length();
+  int first = 0, last = 0;
+  bool is_range = false;
+  
+  for(int cur = 0; cur < input_length; cur++)
```

Comment:
Could you please try with
(1) tokenize with `,`: you can use `strtok` or equivalent
(2) for each token, try change to number, `atoi` or equivalent

Reply:
> (1) tokenize with ,: you can use strtok or equivalent
> (2) for each token, try change to number, atoi or equivalent

I will try them!
- tokenize by istringstream and getline
- change to number using stoi